Doc/distributing/index.rst

```diff
@@ -129,14 +129,10 @@ involved in creating and publishing a project:
 * `Uploading the project to the Python Package Index`_
 * `The .pypirc file`_
 
-.. _Project structure: \
-    https://packaging.python.org/tutorials/packaging-projects/#packaging-python-projects
-.. _Building and packaging the project: \
-   https://packaging.python.org/tutorials/packaging-projects/#creating-the-package-files
-.. _Uploading the project to the Python Package Index: \
-   https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives
-.. _The .pypirc file: \
-   https://packaging.python.org/specifications/pypirc/
+.. _Project structure: https://packaging.python.org/tutorials/packaging-projects/#packaging-python-projects
+.. _Building and packaging the project: https://packaging.python.org/tutorials/packaging-projects/#creating-the-package-files
+.. _Uploading the project to the Python Package Index: https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives
+.. _The .pypirc file: https://packaging.python.org/specifications/pypirc/
```

Comment:
That seems wrong, it looks like directives (which would need double colon) rather than link markup.
Best to have it on one line, that’s reST standard.

Reply:
@hugovk Indeed, I tested and linkcheck doesn't complain if the `\` is removed.

How do we proceed here: One line or in a new line without `\`?

Reply:
One line please!

Reply:
Ok then.

Reply:
> Best to have it on one line, that’s reST standard.

reST specifies that whitespace is ignored specifically to allow breaking over multiple lines! E.g. when lines *must* be wrapped at a certain number of characters.

See https://www.docutils.org/docs/ref/rst/restructuredtext.html#hyperlinks, specifically the following section:

![image](https://user-images.githubusercontent.com/9087854/233499602-b95d6b78-445c-4f7c-a503-d3591bdf73a8.png)

A

Reply:
Ah that’s convenient for reST email for example!

Let me replace «that’s reST standard» with «that’s what’s commonly seen»